chronosphere/pagination/generatepagination/main.go

```diff
@@ -238,7 +241,9 @@ func List{{if .API.RequireUnstable}}Unstable{{end}}{{.Plural}}ByFilter(
 			Context: ctx,
 			PageToken: &nextToken,
 			Slugs: f.Slugs,
+		{{- if .NameFilters}}
 			Names: f.Names,
+		{{- end}}
```

Comment:
optional nit: consider indenting to the same level as code before/after
```suggestion
			{{- if .NameFilters}}
			Names: f.Names,
			{{- end}}
```